lib/sober_swag/compiler/primitive.rb

```diff
@@ -39,6 +39,7 @@ def type_hash
 
       DATE_PRIMITIVE = { type: :string, format: :date }.freeze
       DATE_TIME_PRIMITIVE = { type: :string, format: :'date-time' }.freeze
+      HASH_PRIMATIVE = { type: :object, additionalProperties: true }.freeze
```

Comment:
Little spelling error in the constant...should be HASH_PRIMITIVE (no 'A')

Reply:
good catch!